.rubocop.yml

```diff
@@ -64,3 +67,13 @@ Style/SymbolArray:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
```

Comment:
We should use defaults foreman rules otherwise it won't be suitable

Reply:
Yes. I had a look at foreman rubocop definitions.